collectors/ebpf.plugin/ebpf.d.conf

```diff
@@ -6,9 +6,10 @@
 #  `return : In the `return` mode, the eBPF collector monitors the same kernel functions as `entry`, but also creates
 #            new charts for the return of these functions, such as errors.
 #
-# The eBPF collector also creates charts for each running application through an integration with the `apps plugin`.
-# If you want to disable the integration with `apps.plugin` along with the above charts, change the setting `apps` to
-# 'no'.
+# The eBPF collector also creates charts for each running application through an integration with the `apps plugin`
+# or `cgroup plugin`.
```

Comment:
Please change `cgroup plugin` -> `cgroups plugin` everywhere.

Reply:
or to `cgroups.plugin` and `apps.plugin` as we see in the comment below